src/hdcontroller.ts

```diff
@@ -238,12 +245,13 @@ export class HDProxydClient {
      * creation and its sending.
      */
     private _handleRequest(method: string, stream: Stream|null,
-                           size: number, key: string, log: any,
-                           callback: HDProxydCallback, params: any,
-                           payload: any): void {
-        const headers = params.headers ? params.headers : {};
+        size: number, key: string, log: werelogs.RequestLogger,
+        callback: HDProxydCallback, params: Params,
+        payload: object | undefined): void {
```

Comment:
why reformat this? is this required by eslint?

Reply:
same as here : https://github.com/scality/hdclient/pull/833/files#r1795438326